### PR TITLE
Feat/open buildings from contact

### PR DESCRIPTION
**Pull request by simon-the-shark**
I've been fixing some issues that occured during filling the info in the backend and done the opening links as well (haven't seen your assignemnet @mikolaj-jalocha ) Upsik

- done this map links opening
- changed `plac` to `pl.` to be consistent with buildings
- fixed tab bar transition direction in some cases
- upgraded flutter version to `3.24.0`

### CODE REVIEW

File: lib/features/about_us_view/widgets/contact_section.dart [hunk 1]
```diff
@@ -27,17 +28,17 @@ class ContactSection extends StatelessWidget {
...
   final String url;
   final String icon;
 
   @override
-  Widget build(BuildContext context) {
+  Widget build(BuildContext context, WidgetRef ref) {
     return GestureDetector(
       onTap: () async {
-        await LaunchUrlUtil.launch(url);
+        await ref.launch(url);
```
Comment (mikolaj-jalocha) [inline lib/features/about_us_view/widgets/contact_section.dart:41]: Can we use here `unawaited`?

File: lib/utils/launch_url_util.dart [hunk 2]
```diff
@@ -1,11 +1,36 @@
...
 import "package:url_launcher/url_launcher.dart";
 
-class LaunchUrlUtil {
-  static Future<bool> launch(String uriStr) async {
+import "../features/buildings_view/repository/buildings_repository.dart";
+import "../features/navigator/utils/navigation_commands.dart";
+import "../utils/where_non_null_iterable.dart";
+
+extension LaunchUrlUtilX on WidgetRef? {
+  Future<bool> launch(String uriStr) async {
```
Comment (mikolaj-jalocha) [inline lib/utils/launch_url_util.dart:10]: I like the change here. Universal in all places rn.

**Review by mikolaj-jalocha — APPROVED** [review]
Looks good to me :-) i especially like the  new way of launching` url utils` with help of `ref`. 

I left only one note regarding minor change.

File: lib/features/about_us_view/widgets/contact_section.dart (hunk 1 above)
Comment (simon-the-shark) [reply lib/features/about_us_view/widgets/contact_section.dart:41]: Sure, but also there's nothing after it, so 
```dart
onTap: () async => ref.launch(url),
```
should have similar effect I believe

**Pull request merged**